fdbserver/storageserver.actor.cpp

```diff
@@ -2880,9 +2870,8 @@ ACTOR Future<Void> getKeyValuesAndHopQ(StorageServer* data, GetKeyValuesAndHopRe
 			GetKeyValuesReply _r = wait(
 			    readRange(data, version, KeyRangeRef(begin, end), req.limit, &remainingLimitBytes, span.context, type));
 
-			//			std::cout << "read range done, start hopping" << std::endl;
+			// Hop!!!
```

Comment:
nit: I feel directly changing the std::cout to comment is more reader-friendly than "Hop":
`read range done, start hopping`